src/lib/listCountdowns.js

```diff
@@ -20,6 +21,10 @@ const formatAndNotify = configuration => countdowns => R.pipe(
   message => notify(message)(configuration)
 )(countdowns)
 
+const sortEventsByDate = (events) => {
```

Comment:
you can actually just do

```
const sortEventsByDate = (events) =>  R.sort(
    (e0, e1) => moment(e0.date).isBefore(e1.date) ? -1 : (moment(e0.date).isSame(e1.date) ? 0 : 1), 
    events
)
```

Reply:
D'oh!  I realized when I walked out this morning that I had missed this.  Fixing it ASAP